packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata-v2.service.ts

```diff
@@ -165,50 +166,127 @@ export class ObjectMetadataServiceV2 {
     workspaceId: string;
     isSystemBuild?: boolean;
   }): Promise<ObjectMetadataDTO> {
+    const deletedObjectMetadataDtos = await this.deleteManyObjectMetadatas({
```

Comment:
nit: for me DTO was referring to the transport layer not the data itself

Reply:
Indeed we could add an abstraction layer here to make the `deleteManyObjectMetadatas` return `FlatObjectMetadata` instead of dtos

Reply:
done in https://github.com/twentyhq/twenty/pull/15823